runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/collections/general.ts

```diff
@@ -20,6 +20,8 @@ const general: typeof englishGeneral = {
     'Process.Collection.Tile.MenuItem.Move': 'Przenieś',
     'Process.Collection.Tile.MenuItem.Delete': 'Usuń',
     'Process.Collection.Tile.MenuItem.Edit': 'Edytuj',
+    'Process.Collection.Navigation.Processes.Label': 'Procesy',
+    'Process.Collection.Navigation.Collections.Label': 'Kolekcje',
```

Comment:
A long time ago in RunBotics galaxy far, 
far away....
someone translated collections to "zbiory"

imo unfortunate but let's stick to it